src/app/apps/rup/mapa-camas/views/mapa-camas-capa/mapa-camas-capa.component.ts

```diff
@@ -181,7 +183,11 @@ export class MapaCamasCapaComponent implements OnInit, OnDestroy {
             })
         );
     }
+    verListadoRecursos() {
+        this.listadoRecursos = this.listadoRecursos ? false : true;
+
 
+    }
```

Comment:
```suggestion
    verListadoRecursos() {
        this.listadoRecursos = !this.listadoRecursos;
    }
    
```
Sugerencia: un nombre más apropiado para esta función podría ser `toggleListadoRecursos`

Reply:
O quizás se pueda cambiar el valor en el template, ya que no se realiza otra acción además de esta.